django/contrib/admin/utils.py

```diff
@@ -71,21 +74,7 @@ def quote(s):
 
 def unquote(s):
     """Undo the effects of quote(). Based heavily on urllib.parse.unquote()."""
```

Comment:
Do you think "Based heavily on urllib.parse.unquote()." is still relevant? I imagine the implementation is now quite a bit different (if that was the original intent of the comment).